src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/XmlPropertyFormatter.cs

```diff
@@ -204,7 +212,7 @@ static string SimplifyScalar(object value)
         {
             if (value == null) return null;
 
-            return new XText(value.ToString()).ToString();
+            return new XText(_invalidXMLChars.Replace(value.ToString(), m => "\\u" + ((int)(byte)m.Value[0]).ToString("x4"))).ToString();
```

Comment:
Casting to `byte` here will overflow (16-bit) characters; I think `ushort` is needed

Reply:
Changed!